build/test-aot-compatibility.ps1

```diff
@@ -0,0 +1,38 @@
+$rootDirectory = Split-Path $PSScriptRoot -Parent
```

Comment:
Did you mistakenly add another Powershell script?

Reply:
I wanted to rename the script. Fixed now.